doc/lua_api.txt

```diff
@@ -1730,6 +1730,7 @@ Helper functions
       * nil: return all entries,
       * true: return only subdirectory names, or
       * false: return only file names.
+* `minetest.get_areastore_typenames()`: returns a `areastore_typename -> true` list of possible AreaStore implementations. The mapped value may change in the future to some non-false (and non-nil) value.
```

Comment:
Be consistent: `get_area_store_type_names`

Also, should mods really care about implementation details like this?